MantidQt/CustomInterfaces/test/MuonAnalysisFitFunctionPresenterTest.h

```diff
@@ -223,7 +224,17 @@ class MuonAnalysisFitFunctionPresenterTest : public CxxTest::TestSuite {
     EXPECT_CALL(*m_fitBrowser, setMultiFittingMode(false)).Times(1);
     m_presenter->setMultiFitState(MultiFitState::Disabled);
   }
+  void test_setTFAsymmMode_On() {
```

Comment:
spoke to Nick and he said that with the current implementation that is not possible.